src/main/java/com/gdsc/studyjamapi/common/Constants.java

```diff
@@ -19,5 +19,9 @@ public static class ErrorMessage {
         "The length of the password must be at least 8 characters.";
     public static final String EMAIL_ALREADY_EXISTS = "This email already exists.";
     public static final String NOT_FOUND_MESSAGE_TEMPLATE = "%s not found";
+    public static final String REQUIRED_TITLE = "The title is required.";
+    public static final String REQUIRED_DESCRIPTION = "The task description is required.";
+    public static final String REQUIRED_REQUIREMENT = "The task requirement is required.";
+    public static final String EVENT_NOT_FOUND = "Event not found.";
```

Comment:
No need to create the `EVENT_NOT_FOUND` here. You can reuse the `NOT_FOUND_MESSAGE_TEMPLATE`.